src/controllers/notifications.js

```diff
@@ -1,67 +0,0 @@
-'use strict'
```

Comment:
Ok, well in any case I think we need to implement reliable notifications on the ledger's side